tests/robot-tests/tests/admin_and_public/data-catalogue.robot

```diff
@@ -0,0 +1,177 @@
+    ...    upload-file-test-with-filter.meta.csv
+
+Add data guidance to subjects (second release)
+    user clicks link    Data and files
+    user waits until h2 is visible    Add data file to release
+
+    user clicks link    Data guidance
+
+    user waits until page contains element    id:dataGuidance-dataFiles
+    user waits until page contains accordion section    ${SUBJECT_NAME_3}
+
+    user enters text into data guidance data file content editor    ${SUBJECT_NAME_3}
+    ...    ${SUBJECT_NAME_3} data guidance content
+
+    user enters text into data guidance data file content editor    ${SUBJECT_NAME_4}
+    ...    ${SUBJECT_NAME_4} data guidance content
+
+Save data guidance (second release)
+    user clicks button    Save guidance
+
+Approve release
+    user clicks link    Sign off
+    user approves release for immediate publication
+
+User goes to public Find Statistics page
+    user navigates to find statistics page on public frontend
+
+Verify newly published release is on Find Statistics page
+    user waits until page contains accordion section    %{TEST_THEME_NAME}
+    user opens accordion section    %{TEST_THEME_NAME}
+    user waits until accordion section contains text    %{TEST_THEME_NAME}    ${TOPIC_NAME}
+
+    user opens details dropdown    ${TOPIC_NAME}
+    user waits until details dropdown contains publication    ${TOPIC_NAME}    ${PUBLICATION_NAME}
+    user checks publication bullet contains link    ${PUBLICATION_NAME}    View statistics and data
+    user checks publication bullet contains link    ${PUBLICATION_NAME}    Create your own tables
+    user checks publication bullet does not contain link    ${PUBLICATION_NAME}    Statistics at DfE
+
+User navigates to data-catalogue page
+    user goes to url    %{PUBLIC_URL}/data-catalogue
+    user waits until page contains title caption    Data catalogue
+    user waits until h1 is visible    Browse our open data
+    user waits until page contains    View all of the open data available and choose files to download.
+
+User checks search filters publications properly
+    user enters text into element    id:publicationForm-publicationIdSearch    Pupil
+    user waits until element is visible    testid:Expand Details Section Pupil absence    10
+    user clears element text    id:publicationForm-publicationIdSearch
+
+Choose publiction
+    user opens details dropdown    Test theme
+    user opens details dropdown    ${TOPIC_NAME}
+    user clicks radio    UI tests - data catalogue %{RUN_IDENTIFIER}
+    user clicks button    Next step
+
+Check page displays correct data
+    user waits for page to finish loading
+    user waits until h1 is visible    Browse our open data
+    user checks page contains    Choose a release
+
+    user checks page contains element
+    ...    //*[@data-testid="Radio item for ${RELEASE_NAME} 2021/22"]//span//strong[contains(.,"This is the latest data")]
```

Comment:
This check is a little clunky. No need to be so over-specific. 

Think something like this would be sufficient:
```robot
user checks element contains  testid:Radio item for ${RELEASE_NAME} 2021/22  This is the latest data
```